### PR TITLE
Improved Command Feedback

### PR DESCRIPTION
**Pull request by CodedStingray**

### CODE REVIEW

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 1]
```diff
@@ -40,12 +40,12 @@ public class WorldShaperMessages {
      * @param position The position that has been set
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:40]: add missing params to doc

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 2]
```diff
@@ -55,13 +55,19 @@ public static String positionSetMessage(int index, Vector3i position, boolean ch
      * @param index The index from which the position has been removed
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:57]: add missing params to doc

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 3]
```diff
@@ -55,13 +55,19 @@ public static String positionSetMessage(int index, Vector3i position, boolean ch
...
-    public static String positionRemovedMessage(int index, boolean changed) {
+    public static String positionRemovedMessage(int index, boolean changed, int areaSize) {
         return messageBuilder(MessageLevel.INFO, true)
                 .t("Control position ").a(index + 1)
                 .t(changed ? " removed." : " was already not set.")
+                .t(areaSize >= 0 ? " (%d)".formatted(areaSize) : "")
                 .build();
     }
 
+    public static String actionPerformedMessage(int actionSize) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:76]: add missing doc

File: src/main/java/net/codedstingray/worldshaper/commands/area/operations/CommandCeiling.java [hunk 4]
```diff
@@ -65,6 +67,10 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
             ActionStack playerActionStack = playerData.getActionStack();
             WorldShaper.getInstance().getActionController().performAction(playerActionStack, action);
 
+            player.sendMessage(messageBuilder(ChatMessageFormatter.MessageLevel.INFO, true)
+                    .t("A total of ").a(action.getActionSize()).t(" blocks have been modified").build()
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/commands/area/operations/CommandCeiling.java]: missing period

File: src/main/java/net/codedstingray/worldshaper/commands/clipboard/CommandMove.java [hunk 5]
```diff
@@ -64,6 +66,10 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
             ActionStack playerActionStack = playerData.getActionStack();
             WorldShaper.getInstance().getActionController().performAction(playerActionStack, action);
 
+            player.sendMessage(messageBuilder(MessageLevel.INFO, true)
+                    .t("Area contents stacked ").a(distance).t(" times ").a(direction.name().toLowerCase()).t(".").build()
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/commands/clipboard/CommandMove.java]: stacked not moved?

**Review by ZidArgs — CHANGES_REQUESTED** [review]

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 1 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:40]: added

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 2 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:57]: added

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 3 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:76]: added

File: src/main/java/net/codedstingray/worldshaper/commands/area/operations/CommandCeiling.java (hunk 4 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/commands/area/operations/CommandCeiling.java]: This didn't even belong here and should've been the actionPerformedMessage from the WorldShaperMessages. Fixed it.

File: src/main/java/net/codedstingray/worldshaper/commands/clipboard/CommandMove.java (hunk 5 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/commands/clipboard/CommandMove.java]: fixed

**Review by ZidArgs — APPROVED** [review]

**Pull request merged**